README.md

```diff
@@ -1,34 +1,34 @@
-# Plotly Image Exporter
+# Plotly Orca: Open-source Report Creator App
```

Comment:
Not sure we want that _Open-source Report Creator App_ bit in our title. That was more of a joke. I'd remove it for now.

Reply:
I have made this change.